### PR TITLE
Finished code challenge 06

### PR DESCRIPTION
**Pull request by PotatoGod123**
- [x ] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

### CODE REVIEW

File: challenges/test/lib/src/test/java/code/linkedlisttest/LinkedListTest.java [hunk 1]
```diff
@@ -44,12 +44,63 @@
...
-  @Test public void firstNodeIsHead(){
+  @Test public void testFirstNodeIsHead(){
     LinkedList newList = new LinkedList();
     newList.insert(1);
     newList.insert(2);
     newList.insert(3);
 
     assertEquals(3,newList.firstNode());
   }
+
```
Comment (Mattpet26) [inline challenges/test/lib/src/test/java/code/linkedlisttest/LinkedListTest.java:55]: Good work, test each method with it's own test!
